sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -149,4 +157,79 @@ export class InstitutionController extends BaseController {
   async getUserTypesAndRoles(): Promise<InstitutionUserTypeAndRoleResponseDto> {
     return this.institutionService.getUserTypesAndRoles();
   }
+
+  @Get("/user/:userName")
+  async getInstitutionLocationByUserName(
+    @Param("userName") userName: string,
+  ): Promise<InstitutionUser> {
```

Comment:
Please avoid returning the entire entities directly. Please create some DTO with the relevant information.